src/javascript/JContent/ContentRoute/ContentTitle/ContentTitle.container.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import ContentTitle from './ContentTitle';
+
+const ContentTitleContainer = ({path}) => {
+    const title = path.substring(path.lastIndexOf('/') + 1);
```

Comment:
This will be changed in the future, I needed this component to finish the story so I created most of it, but it should be updated to display a displayable name instead of a system name.
Here is the ticket:
https://jira.jahia.org/browse/BACKLOG-12425